### DISCUSSION TITLE
How to use workflow to implement CRUD operations?

### DISCUSSION THREAD

**Discussion by Line4GitHub** [OrchardCMS/OrchardCore#10883] [Q&A] [answered]
How to use workflow to implement (ContentItem) CRUD operations?
Is there a sample, tutorial for reference?

**Comment by hishamco** [comment]
Could you please elaborate?

**Reply by Line4GitHub** [reply]
1. I'm sorry, for my English is very poor!
2. Thank you for your reply.

**Comment by ns8482e** [comment, marked as answer]
@Line4GitHub There won’t be any straight answer you are looking - unless you elaborate more on scenarios that you are looking.

IMHO You could create an activity depending on your needs - that reads some input and create content item using content manager.

Docs for workflow: https://docs.orchardcore.net/en/latest/docs/reference/modules/Workflows/

**Reply by Line4GitHub** [reply]
Thank you very much for your guidance.
